File: simex_app/app/src/main/java/com/example/simex_app/data/models/LoginResponse.kt
package com.example.simex_app.data.models

import com.google.gson.annotations.SerializedName

data class LoginResponse(
    @SerializedName("id") val id: Int,
    @SerializedName("nombre") val nombre: String,
    @SerializedName("email") val email: String,
    @SerializedName("rolId") val rolId: Int,
    @SerializedName("tipo") val tipo: String,           // "Agente" o "Cliente"
    @SerializedName("nombreRolReal") val nombreRolReal: String? = null
)

File: smiex_api/smiex_api/DTOs/LoginResponse.cs
namespace simex_api.DTOs // Ajusta 'simex_api' al nombre de tu proyecto
{
    public class LoginResponse
    {
        public int Id { get; set; }
        public string Nombre { get; set; } = null!;
        public string Email { get; set; } = null!;
        public int RolId { get; set; }
        public string Tipo { get; set; } = null!;
        public string? NombreRolReal { get; set; }
        // cambiar byte[] por string? /revisar/
        public string? DniFotoFrontal { get; set; }
        public string? DniFotoTrasera { get; set; }
    }
}